.gitignore

```diff
@@ -47,4 +47,4 @@ language/tools/vscode-extension/out/*
 read-badger
 read-protocol-state
 remove-execution-fork
-bootstrap
+# bootstrap
```

Comment:
I think this is meant to ignore the `bootstrap` binary, but conflicts with the `bootstrap` directory at the root level. We could change this line to `cmd/bootstrap/bootstrap` (I believe this is the default binary output location) to only ignore the binary.